app/waf/src/models/profiles/http_compress.model.ts

```diff
@@ -0,0 +1,376 @@
+/**
+ * Copyright 2019 F5 Networks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {model, property} from '@loopback/repository';
+import {CommonEntity} from '../common.model';
+
+@model()
+export class ProfileHTTPCompression extends CommonEntity {
+  @property({
+    type: 'boolean',
+    required: false,
+    default: false,
+    schema: {
+      create: true,
+      update: true,
+      response: true,
+      example: true,
+      openapi: {
+        enum: [true, false],
```

Comment:
Boolean type may not require to enumerate values. Did you notice it did not work?

Reply:
Yes, it doesn't need the openapi validation, deleted.